pkg/api/daemon_example.json

```diff
@@ -0,0 +1,78 @@
+{
+    "kind": "Daemon",
```

Comment:
This is not a good place to put an example. You want this for testing purposes, documentation, or both?

Documentation about Daemon belongs in the cluster admin guide, in docs/admin.